containers/pytorch/training/README.md

```diff
@@ -82,5 +82,15 @@ The PyTorch Training containers come with two different containers depending on
 - **TPU**: To build the PyTorch Training container for Google Cloud TPUs, an instance with at least one TPU available is required to install `optimum-tpu` which is a Python library with Google TPU optimizations for `transformers` models, making its integration seamless.
 
   ```bash
-  docker build -t us-docker.pkg.dev/deeplearning-platform-release/gcr.io/huggingface-pytorch-training-tpu.2.4.0.transformers.4.41.1.py310 -f containers/pytorch/training/tpu/2.4.0/transformers/4.41.1/py310/Dockerfile .
+  docker build -t us-docker.pkg.dev/deeplearning-platform-release/gcr.io/huggingface-pytorch-training-tpu.2.5.1.transformers.4.46.3.py310 -f containers/pytorch/training/tpu/2.5.1/transformers/4.46.3/py310/Dockerfile .
```

Comment:
Does the `docker build` require a specific TPU-enabled environment? I recall it was not required and could be built anywhere, but asking for confirmation.

Reply:
nope, that is not required for now.

Reply:
No, it is not required. I have configured the container to enable all Google-specific features by default in this repository